### ISSUE TITLE
Getting started issues

### ISSUE THREAD

**Issue by cbo-va** [terrazzoapp/terrazzo#349]
I tried to follow the getting started instructions.

But every time I call `npx tz build` I get the following errors:

```
❯ npx tz build
/myproject/node_modules/.bin/tz: line 1: /Applications: is a directory
/myproject/node_modules/.bin/tz: line 2: README.md: command not found
/myproject/node_modules/.bin/tz: line 3: README.md: command not found
/myproject/node_modules/.bin/tz: line 4: README.md: command not found
/myproject/node_modules/.bin/tz: line 5: syntax error near unexpected token `('
/myproject/node_modules/.bin/tz: line 5: ` * Copyright (c) 2021 Drew Powers'
```

I tried on different systems (OSX, Linux) and with different node versions w/o success. Same with switching between yarn and npm.

Before that, I tested cobalt and could use it without those problems.

What am I doing wrong?

**Comment by drwpow** [comment]
Hm I’m not sure what that issue is. Do you see the same issue with `npx tz@latest build`?

Or if you have it installed in `package.json`, what version are you running?

**Comment by drwpow** [comment]
Ah I see the fix in #358 was likely the cause. This should be fixed in `0.2.1` which was just released. Please let me know if you continue to see the problem! (Note that `npx` does cache packages, so you’ll need to install `@latest` locally to see the change)

**Comment by drwpow** [comment]
Unrelated to your original issue, but you made me realize there were quite a few improvements we could make to `tz init`! Just released a new version that can start from an existing design system and even install/update plugins. Would love feedback if you end up using it

![CleanShot 2024-11-29 at 08 26 43](https://github.com/user-attachments/assets/cfdca8f3-eac8-46ce-b7a2-8a58d5465181)
